tests/udf/test_coords.py

```diff
@@ -0,0 +1,171 @@
+import numpy as np
+
+from libertem.udf import UDF
+from libertem.io.dataset.memory import MemoryDataSet
+
+from utils import _mk_random
+
+
+class test_UDF_by_partition(UDF):
```

Comment:
The UDFs shouldn't be named like test cases - `pytest` will actually complain if they have a `Test` prefix and aren't actually `unittest.TestCase` subclasses. Names should also follow pep8, so camel case for classes, without underscores.